File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int n = nums.length;
        
        //upper pointer/variable
        
        int i=0;
        
    
        //lower pointer 
        for(int j=0;j<n;j++){
            
            //differet elements
            
            if(nums [i]  != nums[j]){
                i++;
                nums[i] = nums[j];
            }
        }
        
        return i+1;
        
        
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int n = nums.size();
        
        int first = 0;
        
        for(int s=0; s<n; s++){
            
            if(nums[first] != nums[s]){
                first++;
                nums[first] = nums[s];
            }
        }
        
        return first+1;
    }
};